packages/volto/locales/volto.pot

```diff
@@ -85,8 +85,8 @@ msgid "Add (object list)"
 msgstr ""
 
 #. Default: "To make new add-ons show up here, add them to your configuration, build, and restart the server process. For detailed instructions see"
-#: components/manage/Controlpanels/AddonsControlpanel
-msgid "Add Addons"
+#: components/manage/Controlpanels/Addons/AddonsControlpanel
+msgid "Add Add-ons"
```

Comment:
Existing `msgid`s should not be changed, even though they are not correctly hyphenated, as it may break stuff. See https://github.com/plone/volto/issues/6159. Please revert any existing `msgid`s that you changed.

You may change `Default` or `msgstr`, or add or edit components.